src/main/jbake/content/using-git.ad

```diff
@@ -0,0 +1,178 @@
+    git push origin opennlp-xxxx # notice pushing to **origin** not **apache**
+
+Once you are ready to commit to the Apache remote you can merge and push them directly, or better yet create a
+pull request (PR). 
+
+## How to create a PR (committers)
+
+Push your branch to GitHub:
+
+    git checkout opennlp-xxxx
+    git push origin opennlp-xxxx
+
+Go to your opennlp-xxxx branch on GitHub. Since you forked it from GitHub's apache/opennlp it will default any PR to
+go to apache/master. 
+
+* Click the green "Compare, review, and create pull request" button. 
+* You can edit the _to_ and _from_ for the PR if it is not correct. The "base fork" should be apache/opennlp unless
+you are collaborating separately with one of the committers on the list. The "base" will be master. Do not submit a
+PR to one of the other branches unless you know what you are doing. The "head fork" will be your forked repo and the
+"compare" will be your opennlp-xxxx branch. 
+* Click the "Create pull request" button and name the request "OPENNLP-XXXX" (uppercase). This will connect the
+comments of the PR to the mailing list and JIRA comments.
+* From now on the PR lives on GitHub's apache/opennlp. You can use the commenting UI there. 
+* If you are looking for a review or sharing with someone else say so in the comments but do not worry about 
+automated merging of your PR -- you will have to do that later. The PR is tied to your branch so you can respond to 
+comments, fix code, and commit from your local repository. They will appear on the PR page and be mirrored to JIRA 
+and to the mailing list. 
+
+When you are satisfied and want to push it to Apache's remote repository proceed with **Merging a PR**
+
+## How to create a PR (contributors)
+
+Create pull requests: <<anchor-1,[1]>>. 
+
+Pull requests are made to apache/opennlp repository on GitHub. In the GitHub UI you should pick the master 
+branch to target the PR as described for committers. This will be reviewed and commented on so the merge is 
+not automatic. This can be used for discussing a contributions in progress.
+
+## Merging a PR (yours or contributors) 
+
+Start with reading <<anchor-2,[2]>> (merging locally). 
+
+Remember that pull requests are equivalent to a remote GitHub branch with potentially a multitude of commits. 
+In this case it is recommended to squash remote commit history to have one commit per issue, rather 
+than merging in a multitude of contributor's commits. In order to do that, as well as close the PR at the 
+same time, it is recommended to use **squash commits**.
+
+Merging pull requests is equivalent to a "pull" of a contributor's branch:
+
+    git checkout master      # switch to local master branch
+    git pull apache master   # fast-forward to current remote HEAD
+    git pull --squash https://github.com/cuser/opennlp cbranch  # merge to master 
+
+`--squash` ensures all PR history is squashed into single commit, and allows committer to use his/her own
```

Comment:
@kinow @kottmann I think it might be worth considering using `git am --signoff` on top of this. So contributor rebases and squash against master just prior to merge (sometimes having the multiple commits during review is helpful) and then the merge by a committer is with `--signoff` which ensures the contributor gets credit, but also the git history reflects who signed off and performed the merge.